src/DropdownMenu/Menu.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import cx from '../utils/classnames';
+import { props, t, ReactChildren } from '../utils';
+import partial = require('lodash/partial');
+import FlexView from 'react-flexview';
+import Divider from '../Divider/Divider';
+
+export namespace MenuProps {
+  export type Option = {
+    type: 'title' | 'item' | 'divider',
+    title?: any, // TODO: t.ReactChildren
+    metadata?: any,
```

Comment:
any?

Reply:
mmm... `metadata` is `t.Any` even in `tcomb` model, but since it's used like this: 
```
<FlexView grow shrink={false} className='menu-item-metadata' hAlignContent='right'>
   {option.metadata}
</FlexView>
```
a `JSX.Element` seems more appropriate.